catalog/pages/inputs/HybridSelectExample.js

```diff
@@ -42,8 +48,8 @@ export default class HybridSelectExample extends Component {
           >
             Option Two
           </HybridOption>
-          <HybridOption value="2" index={2} optionText="Option Three">
-            {/* Add prop optionText while children is not plain text */}
+          {/* Add prop optionText while children is not plain text */}
+          <HybridOption value="2" index={2} optionText=" Option Three">
```

Comment:
Remove extra space from **optionText** string.

Reply:
This space was intentional. Because this will add space between label and selected value. It will be controlled by used.